pyseir/ensembles/ensemble_runner.py

```diff
@@ -2,19 +2,31 @@
 import logging
 import os
 import numpy as np
+import us
 import json
+from enum import Enum
 import copy
 from collections import defaultdict
 from functools import partial
 from multiprocessing.pool import Pool
 from pyseir.models.seir_model import SEIRModel
 from pyseir.parameters.parameter_ensemble_generator import ParameterEnsembleGenerator
-from pyseir.models.suppression_policies import generate_empirical_distancing_policy
+from pyseir.models.suppression_policies import generate_empirical_distancing_policy, generate_covidactnow_scenarios
 from pyseir import OUTPUT_DIR
 from pyseir import load_data
 from pyseir.inference import fit_results
 from pyseir.reports.county_report import CountyReport
 
+from libs.datasets import JHUDataset
+from libs.datasets import FIPSPopulation
+from libs.datasets.dataset_utils import AggregationLevel
+timeseries = JHUDataset.local().timeseries()
+
+
+class RunMode(Enum):
+    DEFAULT = 'default'  # 4 basic scenarios.
+    CAN_BEFORE = 'can-before' # 4 basic scenarios.
```

Comment:
What's the difference here?

Reply:
Oops. Fixed the docstring.   This will be moved to config files soon, but currently the run modes are different parameter sets.